src/components/Book.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book }) => {
```

Comment:
- [ ] Hi @williamscch, Based on the requirements of this project, the button for removing books should be present in this component. This button shouldn't remove the book yet. This is just a placeholder. Kindly implement this requirement and move a step toward approval.

Reply:
Hey, yes that's true thanks for remembering me.